CMakeLists.txt

```diff
@@ -140,18 +165,30 @@ if(UNIX)
         string(REGEX REPLACE "([^;]+)" "${ICU_LIBRARY_DIR}/lib\\1.a" ICU_LIBRARIES "${ICU_LIBRARY_NAMES}")
     endif()
 else()
-    set(BOOST_ROOT       ${IFCOPENSHELL_ROOT}/deps/boost_1_67_0)
-    set(BOOST_LIBRARYDIR ${IFCOPENSHELL_ROOT}/deps/boost_1_67_0/stage/vs${MSVC_YEAR}-${WIN_ARCH}/lib)
+	if (${ifcopenshell_major_version} STREQUAL "0.7")
+		SET(HDF5_INCLUDE_DIR ${IFCOPENSHELL_ROOT}/_deps-vs${MSVC_YEAR}-${WIN_ARCH}-installed/HDF5-1.8.22-win64/include)
+		SET(HDF5_LIBRARY_DIR ${IFCOPENSHELL_ROOT}/_deps-vs${MSVC_YEAR}-${WIN_ARCH}-installed/HDF5-1.8.22-win64/lib)
+		
+		SET(HDF5_LIBRARIES 
+			"${HDF5_LIBRARY_DIR}/libhdf5_cpp.lib"
+			"${HDF5_LIBRARY_DIR}/libhdf5.lib"
+			"${HDF5_LIBRARY_DIR}/libzlib.lib"
+			"${HDF5_LIBRARY_DIR}/libsz.lib"
+			"${HDF5_LIBRARY_DIR}/libaec.lib")
+	endif()
+	
+    set(BOOST_ROOT       ${IFCOPENSHELL_ROOT}/_deps/boost_1_74_0)
+    set(BOOST_LIBRARYDIR ${IFCOPENSHELL_ROOT}/_deps/boost_1_74_0/stage/vs${MSVC_YEAR}-${WIN_ARCH}/lib)
 	
     set(IFC_INCLUDE_DIR  ${IFCOPENSHELL_ROOT}/src)
-    set(IFC_LIBRARY_DIR  ${IFCOPENSHELL_ROOT}/build-vs${MSVC_YEAR}-x86/${CMAKE_BUILD_TYPE})
+    set(IFC_LIBRARY_DIR  ${IFCOPENSHELL_ROOT}/_build-vs${MSVC_YEAR}-${WIN_ARCH}/${CMAKE_BUILD_TYPE})
     
     string(REGEX REPLACE "([^;]+)" "${IFC_LIBRARY_DIR}/\\1.lib" IFC_LIBRARIES "${IFC_LIBRARY_NAMES}")
 
     if (${ifcopenshell_major_version} STREQUAL "0.5")
         set(ICU_LIBRARY_NAMES icuuc icudt)
-        set(ICU_LIBRARY_DIR  ${IFCOPENSHELL_ROOT}/deps-vs${MSVC_YEAR}-x86-installed/icu/lib)
-        set(ICU_INCLUDE_DIR  ${IFCOPENSHELL_ROOT}/deps-vs${MSVC_YEAR}-x86-installed/icu/include)
+        set(ICU_LIBRARY_DIR  ${IFCOPENSHELL_ROOT}/_deps-vs${MSVC_YEAR}-${WIN_ARCH}-installed/icu/lib)
```

Comment:
Add the time of the 0.5 build script the dir was still called `deps` not `_deps` can you change that back?